components/core/src/util/docker.rs

```diff
@@ -0,0 +1,42 @@
+use crate::{env,
+            error::{Error,
+                    Result},
+            fs::find_command};
+use std::{path::PathBuf,
+          process::Command};
+
+const DOCKER_CMD: &str = "docker";
+const DOCKER_CMD_ENVVAR: &str = "HAB_DOCKER_BINARY";
+
+pub fn command() -> Result<PathBuf> {
```

Comment:
I was changing to return a `Command` since on the surface that seemed like the right thing to do but ended up going with `command_path` because we may need to create several `command` instances in a single operation but only want to run through this logic to get the path once.